.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         run: yarn
       - name: Run test ${{ matrix.test-name }}
         run: yarn test:${{ matrix.test-name }}
+        working-directory: packages/default-storage-backend
```

Comment:
I thought more about TYPE_OF_STORAGE-storage-backend convention. We already have other storage called `sqlite-storage-backend`. Besides, I don't think we will have more storages in this monorepo anyway

Reply:
It would look a lot nicer in the directory if we stick to the storage-<category>-<implementation> convention though:
```
storage-backend-default   # What we have in 1.x
storage-backend-mmkv      # Maybe?
storage-backend-os-prefs  # NSUserDefaults (iOS) or SharedPreference (Android)
storage-backend-sqlite    # SQLite
storage-core              # JS
storage-types             # TypeScript interfaces
```
But I suppose we could also change this later, so I'll let you decide.